README.md

```diff
@@ -29,7 +29,7 @@ To get a local copy up and running follow these simple example steps.
 * Open the terminal
 * Run ```cd \path\to\where\you\want\your\local\copy```
 * Clone the repo
-  * Run ```git clone https://github.com/Trictonicmp/to-do-list.git```
+  * Run ```git clone https://github.com/Trictonicmp/Leaderboard.git```
   * Run ```cd README/```
```

Comment:
- [ ] In your `README` file, kindly update the getting started section. This should be an example of how you may give instructions on setting up your project locally. Your project is not running the way you mentioned here. :+1: